plugins/zoom/icon_zoom/tasks/monitor_sign_in_out_activity/task.py

```diff
@@ -0,0 +1,168 @@
+import insightconnect_plugin_runtime
+from .schema import (
+    MonitorSignInOutActivityInput,
+    MonitorSignInOutActivityOutput,
+    MonitorSignInOutActivityState,
+    Component,
+)
+
+# Custom imports below
+from datetime import datetime, timedelta
+from icon_zoom.util.event import Event
+
+
+class MonitorSignInOutActivity(insightconnect_plugin_runtime.Task):
```

Comment:
I wonder if we could move all the methods except `run` to utils for example 😄?

Reply:
I'd like to keep them within the task.py, since they're all pretty specific to the task (I don't think they are too useful outside of the task right now).